python/feathub/processors/flink/table_builder/tests/test_kafka_utils.py

```diff
@@ -101,7 +101,7 @@ def test_kafka_source(self):
                 expected_options, dict(flink_table_descriptor.get_options())
             )
 
-            source.is_bounded = True
+            source._is_bounded = True
```

Comment:
It is preferred to use FeatHub public API in tests so that the tests mimic how user code interacts with FeatHub.

How about we use `source.get_bounded_view()` here?